onyo/commands/tests/test_new.py

```diff
@@ -86,6 +86,98 @@ def test_new_top_level(repo: OnyoRepo) -> None:
     fsck(repo)
 
 
+@pytest.mark.repo_dirs(*directories)
+def test_new_from_root_with_absolute_paths(repo: OnyoRepo) -> None:
+    """
+    Test that `onyo new --path <path>` can create an asset with absolute paths
+    while in the root of the repository.
+    """
+    file = 'laptop_apple_macbookpro.faux'
+
+    for d in directories:
+        # build an absolute path for a new asset
+        new_file = Path(f"{d}/{file}").resolve()
+
+        ret = subprocess.run(['onyo', 'new', '--yes', '--path', new_file],
+                             capture_output=True, text=True)
+
+        # verify correct output
+        assert "The following will be created:" in ret.stdout
+        assert str(d) in ret.stdout
```

Comment:
```suggestion
```

You assert that `newfile` is showing up already. `new_file` is an absolute path containing `d`. So, this assertion here is superfluous.

Reply:
I am not sure, that this is actually superfluous, though it does rather test the output. And I think, especially with the refactoring of output that we plan to centralize printing in onyo, that to have this is not necessarily a bad thing for later.

Reply:
Woops, just saw the code, I literally check `assert str(new_file) in ret.stdout` so I do what I want to do there indeed already, you were right and I removed those lines! :)